src/components/Scene.tsx

```diff
@@ -1,7 +1,11 @@
+import { useAppSelector } from '@/hooks/use-redux.ts'
```

Comment:
the `.ts` file extension is unnecessary
```suggestion
import { useAppSelector } from '@/hooks/use-redux'
```